doc/HWReleaseProcess.md

```diff
@@ -19,10 +19,11 @@
 
 ## Major Releases
 
-1. The RTL team will consume the **release collateral** generated by the **Nightly Workflow** and verify it in the DV environment. The verification should include RTL synthesis using the chosen tool from the Caliptra WG.
-2. Upon success of the DV verification a new release will be made on the **caliptra** repo
-    -  A ZIP file containing the release collateral will be included as part of the release.
-4. The **caliptra-rtl** and **caliptra-sw** repos will be tagged with a **release** version tag.
+1. Before finalizing an RTL release, the hardware team will follow the procedure described in [CaliptraReleaseChecklist.md](https://github.com/chipsalliance/caliptra-rtl/blob/main/docs/CaliptraReleaseChecklist.md).
+  - Upon finalizing all release checklist items, the hardware team will generate a release tag on the RTL **main** branch, following the naming convention `v<major>.<minor>`.
+  - When this is completed, the RTL team will create a new branch following the naming convention `patch_v<major>.<minor>`. This branch will exactly match the contents of the major release tag. Any future fixes that must be applied as patches to this release version will be committed to the `patch_v<major>.<minor>` branch. Branch **main** will continue to receive updated code for the subsequent planned release, and is not guaranteed to contain a valid release except as tagged.
+2. The software team will follow the [submodule update steps](#RTL-Submodule-Pointer-Update) to revise the RTL submodule pointer to consume the code from branch `patch_v<major>.<minor>` instead of from branch **main**. 
+3. The **caliptra-sw** repo will be tagged with a **release** version tag. As a result of the submodule flow, this release will contain a snapshot of the code from **caliptra-rtl** repo that has already been tagged with the release version.
```

Comment:
`of the code from **caliptra-rtl** repo that has already been tagged`-->`of the code from the **caliptra-rtl** repo that was already tagged`